.github/workflows/codegen.yaml

```
@@ -40,17 +40,18 @@ jobs:
         run: |
           make bundle
       - name: Ensure there is no diff in bundle
+        # --ignore-matching-lines='.*createdAt:.*' added to ignore bundle differences of timestamp that is generated by upgraded operator-sdk
         run: |
-          git diff --exit-code -- .
+          git diff --ignore-matching-lines='.*createdAt:.*'
       - name: Run make generate
         run: |
           make generate
       - name: Ensure there is no diff in generated assets
         run: |
-          git diff --exit-code -- .
+          git diff --ignore-matching-lines='.*createdAt:.*'
```

```suggestion
          git diff --ignore-matching-lines='.*createdAt:.*' --exit-code -- .
```
Same as previous comment